component/src/main/java/org/wso2/extension/siddhi/io/http/sink/util/HttpSinkUtil.java

```diff
@@ -198,4 +261,74 @@ public String getContentType(String mapType, List<Header> headersList) {
             }
         }
     }
+
+    /**
+     * Get port from listenerUrl.
+     *
+     * @param senderUrl the listener URL.
+     * @return port of the listener URL.
+     */
+    public static String getScheme(String senderUrl) {
+        URL aURL;
+        try {
+            aURL = new URL(senderUrl);
+        } catch (MalformedURLException e) {
+            throw new SiddhiAppCreationException("SenderUrl is not in a proper format ", e);
+        }
+        return aURL.getProtocol();
+    }
+
+    public static Map<String, Object> populateClientConnectionConfiguration(
+            Map<String, Object> clientConnectionConfigurationList) {
+        Map<String, Object> properties = new HashMap<>();
+        Map<String, TrpPropertyTypes> tryMap = trpPropertyTypeMap();
+        clientConnectionConfigurationList.forEach((key, value) -> {
+            switch (tryMap.get(key).name()) {
+                case "BOOLEAN":
+                    properties.put(key, Boolean.valueOf((String) value));
+                    break;
+                case "STRING":
+                    properties.put(key, value);
+                    break;
+                case "INTEGER":
+                    properties.put(key, Integer.valueOf((String) value));
+                    break;
+                case "DOUBLE":
+                    properties.put(key, Double.valueOf((String) value));
+                    break;
+                default:
+                    log.error("Transport property is no defined.");
```

Comment:
Please log the property too.

Reply:
Fixed in 6bf095af9d7a4c00220069badb50517cec553139